File: apps/web/src/lib/devAccounts.ts
import type { Role } from './types';

/** Contas de teste — espelho de services/core/src/dev_accounts.py. Apenas para desenvolvimento. */
export const DEV_ACCOUNTS: { email: string; password: string; first: string; last: string; role: Role }[] = [
  { email: 'admin@gdesigner.school', password: 'Admin123!', first: 'Admin', last: 'AOS', role: 'admin' },
  { email: 'coordenacao@gdesigner.school', password: 'Coord123!', first: 'Marta', last: 'Fernandes', role: 'coordinator' },
  { email: 'docente@gdesigner.school', password: 'Docente123!', first: 'Paulo', last: 'Neto', role: 'teacher' },
  { email: 'estudante@gdesigner.school', password: 'Aluno123!', first: 'Ana', last: 'Kianda', role: 'student' },
  { email: 'secretaria@gdesigner.school', password: 'Secret123!', first: 'Rosa', last: 'Chipenda', role: 'secretary' },
  { email: 'biblioteca@gdesigner.school', password: 'Biblio123!', first: 'João', last: 'Sakala', role: 'librarian' },
  { email: 'rh@gdesigner.school', password: 'Rh123!', first: 'Lúcia', last: 'Mbala', role: 'hr' },
  { email: 'encarregado@gdesigner.school', password: 'Pai123!', first: 'Domingos', last: 'Kianda', role: 'parent' },
  { email: 'convidado@gdesigner.school', password: 'Guest123!', first: 'Visitante', last: 'AOS', role: 'guest' },
];

File: services/core/src/dev_accounts.py
"""Contas de teste (apenas ENVIRONMENT=development|test). Uma por papel.
Mesma lista é espelhada no frontend em apps/web/src/lib/devAccounts.ts."""

DEV_PASSWORD_SUFFIX = "123!"

DEV_ACCOUNTS = [
    # email, first, last, role, password
    ("admin@gdesigner.school", "Admin", "AOS", "admin", "Admin123!"),
    ("coordenacao@gdesigner.school", "Marta", "Fernandes", "coordinator", "Coord123!"),
    ("docente@gdesigner.school", "Paulo", "Neto", "teacher", "Docente123!"),
    ("estudante@gdesigner.school", "Ana", "Kianda", "student", "Aluno123!"),
    ("secretaria@gdesigner.school", "Rosa", "Chipenda", "secretary", "Secret123!"),
    ("biblioteca@gdesigner.school", "João", "Sakala", "librarian", "Biblio123!"),
    ("rh@gdesigner.school", "Lúcia", "Mbala", "hr", "Rh123!"),
    ("encarregado@gdesigner.school", "Domingos", "Kianda", "parent", "Pai123!"),
    ("convidado@gdesigner.school", "Visitante", "AOS", "guest", "Guest123!"),
]
